packages/aurora-feed/index.js

```diff
@@ -57,25 +61,26 @@ class Feed extends React.Component {
     this.onChange = this.onChange.bind(this);
     this.searchCard = this.searchCard.bind(this);
     this.addSavedNotes = this.addSavedNotes.bind(this);
+    this.onDelete = this.onDelete.bind(this);
     loadNotes(this.addSavedNotes);
   }
 
-  addSavedNotes(editorStates) {
-    editorStates.forEach(state => {
-      this.addCard(state);
+  addSavedNotes(notes) {
+    notes.forEach(note => {
+      this.addCard(note);
     });
   }
 
-  addCard(editorState) {
+  addCard(note) {
     // Don't add a note if it doesn't exist. AUR-20
-    const text = editorState.getCurrentContent().getPlainText();
+    const text = note["editorState"].getCurrentContent().getPlainText();
```

Comment:
Don't use the string format whenever possible. It can lead to dangerous things in the future where people try to dynamically access pieces of the JSON like this:

``` js
note["editorState" + someNumber.getSomething()] // Ugly and dangerous
```

A better style that avoids bugs is this:
``` js
note.editorState.getCurrentContent().getPlainText();
```

That does the same thing without encouraging danger 🚨🚨
🚨